app/utils/print/sections/AnswerRowConverter.scala

```diff
@@ -193,6 +193,21 @@ class AnswerRowConverter @Inject()(checkAnswersFormatters: CheckAnswersFormatter
     }
   }
 
+  def numberOfBeneficiariesQuestion[T](query: Gettable[T],
+                        userAnswers: UserAnswers,
+                        labelKey: String,
+                        messageArg: String = "")
+                       (implicit messages: Messages, rds: Reads[T]): Option[AnswerRow] = {
+
+    userAnswers.get(query) map {x =>
+      AnswerRow(
+        messages(s"$labelKey.checkYourAnswersLabel", messageArg),
+        HtmlFormat.escape(messages(s"numberOfBeneficiaries.${x.toString}")),
```

Comment:
Very minor but I believe this line can just be `HtmlFormat.escape(messages(s"numberOfBeneficiaries.$x")),`
When a value is used in a string interpolator the `toString` method is called on it by default